java/samples/sample-code/src/main/java/com/microsoft/semantickernel/samples/syntaxexamples/Example56_TemplateMethodFunctionsWithMultipleArguments.java

```diff
@@ -0,0 +1,88 @@
+package com.microsoft.semantickernel.samples.syntaxexamples;
+
+import com.azure.ai.openai.OpenAIAsyncClient;
+import com.azure.ai.openai.OpenAIClientBuilder;
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.core.credential.KeyCredential;
+import com.microsoft.semantickernel.Kernel;
+import com.microsoft.semantickernel.chatcompletion.ChatCompletionService;
+import com.microsoft.semantickernel.orchestration.PromptExecutionSettings;
+import com.microsoft.semantickernel.orchestration.contextvariables.KernelArguments;
+import com.microsoft.semantickernel.plugin.KernelPluginFactory;
+import com.microsoft.semantickernel.samples.plugins.text.TextPlugin;
+import com.microsoft.semantickernel.semanticfunctions.KernelFunctionFromPrompt;
+import com.microsoft.semantickernel.semanticfunctions.KernelPromptTemplateFactory;
+import com.microsoft.semantickernel.semanticfunctions.PromptTemplateConfig;
+
+public class Example56_TemplateMethodFunctionsWithMultipleArguments {
+
+    private static final boolean USE_AZURE_CLIENT = Boolean.parseBoolean(
+        System.getenv("USE_AZURE_CLIENT"));
+    private static final String CLIENT_KEY = System.getenv("CLIENT_KEY");
```

Comment:
I thought we were still going with AZURE_OPENAI_KEY and OPENAI_KEY (or something like that). I thought the point of USE_AZURE_CLIENT was to differentiate if I have both AZURE_OPENAI_KEY and OPENAI_KEY defined.